.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
We haven't usually been checking this sort of thing in, in favor of people setting that in their global config.  Was this added by mistake?

Reply:
@davidlehn np, I can remove it.